internal/pkg/describe/status_describe.go

```diff
@@ -197,6 +207,21 @@ func (s *ecsStatusDescriber) ecsServiceAutoscalingAlarms(cluster, service string
 	if err != nil {
 		return nil, fmt.Errorf("get auto scaling CloudWatch alarms: %w", err)
 	}
+	for i := range alarms {
+		alarms[i].Type = autoscalingAlarmType
```

Comment:
Why are we setting all the alarm types to be `autoscalingAlarmType`?

Reply:
So it's easier to distinguish between Auto Scaling alarms and alarms for deployment rollbacks.